src/app/model/student.ts

```diff
@@ -3,5 +3,5 @@ export interface Student {
   firstName: string;
   lastName: string;
   email: string;
-  active: boolean;
+  marketingEnabled: boolean;
```

Comment:
to jest status aktywny / nieaktywyny itd. ? Czemu akurat tak się nazywa

Reply:
to jest pole z modelu z backendu, taki tam ustaliliśmy więc na taki zmieniłem (we wcześniejszej wersji miałem active, potem stwierdziliśmy, że rezygnujemy z tego active ale za to mamy marketingEnabled

Reply:
Masz racje jest takie pole, ale do czego ma służyć to nie wiem, niby nie było żadnego PR jeszcze do modelu